frontend/src/pages/modelServing/screens/projects/usePrefillModelDeployModal.ts

```diff
@@ -100,7 +100,7 @@ const usePrefillModelDeployModal = (
         if (recommendedConnections.length === 0) {
           setCreateData('storage', {
             awsData: prefilledAWSData,
-            dataConnection: '',
```

Comment:
@manaswinidas in the existing ModelDeployPrefillInfo type, can we either rename `modelArtifactStorageKey` to something like `initialConnectionName` or move it into the `modelRegistryInfo` sub-object there? The current name is model registry specific - I would have renamed it in #4002 but didn't want to cause conflicts with your branch here.

Reply:
Renamed `modelArtifactStorageKey` to `initialConnectionName`